viewing_party/party.py

```diff
@@ -0,0 +1,199 @@
+def create_movie(title, genre, rating):
+    if title and genre and rating:
+        return {
+                "title": title,
+                "genre": genre,
+                "rating": rating
+        }
+    else:
+        return None
+
+def add_to_watched(user_data , movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data , movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data , title):
+# checks if movie title is in watchlist
+# if it's remove from watchlist
+# and add movie to watched list
+#  return the user_data
+# if movie is not in the watchlist return the user_data
+    for i in range(len(user_data["watchlist"])):
+        if user_data["watchlist"][i]["title"] == title:
+            movie = user_data["watchlist"].pop(i)
+            return add_to_watched(user_data,movie)
+
+    return user_data
+
+def get_watched_avg_rating(user_data):
+    sum_num = 0
+    movie_list = user_data["watched"]
+    for movie in movie_list:
+        sum_num = sum_num + movie["rating"]
```

Comment:
Good job accumulating the sum. Note that we will commonly see this written as
```suggestion
        sum_num += movie["rating"]
```